client/npm-shrinkwrap.json

```diff
@@ -18,32 +18,26 @@
       "resolved": "https://registry.npmjs.org/accepts/-/accepts-1.2.13.tgz"
     },
     "acorn": {
-      "version": "3.0.4",
-      "from": "acorn@>=3.0.4 <4.0.0",
-      "resolved": "https://registry.npmjs.org/acorn/-/acorn-3.0.4.tgz"
+      "version": "3.1.0",
```

Comment:
UPGRADE NOTE: Do not worry about the diffs in this file. You will update this when you run `npm i` and maybe `npm shrinkwrap`. You will probably `rm -rf node_modules`. All those commands are to be run from the `/client` dir.